src/shared/components/person/settings.tsx

```diff
@@ -119,6 +120,8 @@ interface SettingsState {
   searchInstanceOptions: Choice[];
   isIsomorphic: boolean;
   show2faModal: boolean;
+  importOrExportLoading: "importing" | "exporting" | null;
```

Comment:
Why not have this as an `importAccountRes` and `exportAccountRes` like the other calls above, which already has a loading state. Seems more consistent.

Reply:
Using a response object is somewhat awkward here as the import and export responses don't have types in the js client.

Reply:
`RequestState<any>` still has the loading and success states, no need to make custom ones then.